### ISSUE TITLE
[QUESTION] Mod Permission

### ISSUE THREAD

**Issue by 365Tito** [FlansMods/FlansMod#1381]
Hi! I have been a user of Flan's mod since it's earliest days. I've always liked the way this mod handled guns and crafting of guns.
I was sad when I saw that it won't be updated for newer versions. Since I know java, I would like to ask for permission to port your mod to 1.16.5.

**Comment by Trhod17** [comment]
I'm going to start this off by saying im not affiliated with flans mod and im not a lawyer but according to its license [on the flansmod website](https://flansmod.com/license) and [on github](https://github.com/FlansMods/FlansMod/blob/1.12.2/LICENSE.txt) which contains the same elements as [this one](https://creativecommons.org/licenses/by-nc-sa/3.0/) as long as you give credit and upload your updated version using the same license you should be all good.

**Comment by 365Tito** [comment]
@Trhod17 yes, I know, but still, I would like to ask @jamioflan for permission first, just to be sure. It's good practice anyways.

**Comment by recolic** [comment]
Usually, it doesn't require another dedicated permission to fork an open-source project, since this project already has a clear license.

**Comment by JLaH46oKC** [comment]
@365Tito Hi, I would really like to see flansmod update to 1.16 as this is the version of my server I would like to use the flans mod for. but unfortunately there will be no new updates by Flan, so I hope you will not give up your idea, please contact me on discord **гадюшник#5964**,

**Comment by Jeilds** [comment]
I would also like to see a 1.16.5 flans mod pls make it lol
